.github/workflows/fake-release.yml

```diff
@@ -0,0 +1,77 @@
+name: "Fake Release Test (Linux Integration)"
+
+# This workflow tests the Linux desktop integration with release process
+# Triggers only on the specific branch to test the reusable workflow
+on:
+  push:
+    branches: [ "micn/linux-desktop-app" ]
```

Comment:
do we still want to keep this around? if so maybe just name it a dedicated `test/linux-desktop-app` branch to use?